docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_use_custom_expectations.md

```diff
@@ -24,7 +21,9 @@ This guide will walk you through the process of utilizing Custom Expectations, w
 
 </Prerequisites>
 
-## Steps
+There are slight differences between working with custom Expectations and those contributed to GX.
```

Comment:
```suggestion
Custom Expectations and those contributed to GX are different.
```
Why is this important? Why should a user care?

Reply:
because the way they are imported/referenced is different. There are tabs in the doc that describe the differences